### ISSUE TITLE
Error of Network Traffic Model Forecasting

### ISSUE THREAD

**Issue by MengleiZhang** [intel-analytics/analytics-zoo#4512]
I got the following error when running the following example: [network_traffic_model_forecasting.ipynb](https://github.com/intel-analytics/analytics-zoo/blob/master/pyzoo/zoo/chronos/use-case/network_traffic/network_traffic_model_forecasting.ipynb). However, when I tried the older version of this file (before the update on Aug 10), there was no error. Can you check?

I installed the analytics-zoo using `pip install analytics-zoo`.

---
forecaster = LSTMForecaster(past_seq_len=x_train.shape[1],
                            input_feature_num=x_train.shape[-1],
                            output_feature_num=y_train.shape[-1],
                            hidden_dim=16,
                            layer_num=2,
                            lr=0.001)
----
TypeError                                 Traceback (most recent call last)
/tmp/ipykernel_3636247/1497806761.py in <module>
      8                             hidden_dim=16,
      9                             layer_num=2,
---> 10                             lr=0.001)

TypeError: __init__() got an unexpected keyword argument 'past_seq_len'

**Comment by TheaperDeng** [comment]
Hi,

We updated our API and had some improvement on forecaster since last release and the notebook on master branch reflect the changes. There are two options to make things work.

**1.** upgrade the analytics-zoo to our weekly-built version by

```
pip install --pre --upgrade analytics-zoo[automl]
```

and continue using the notebooks on master branch

related **latest** doc:

user guide: https://analytics-zoo.readthedocs.io/en/latest/doc/Chronos/Overview/chronos.html

api doc: https://analytics-zoo.readthedocs.io/en/latest/doc/PythonAPI/Chronos/index.html

**2.** Continue using analytics-zoo 0.11.0 released version and notebooks on tag 0.11.0: https://github.com/intel-analytics/analytics-zoo/blob/v0.11.0/pyzoo/zoo/chronos/use-case/network_traffic/network_traffic_model_forecasting.ipynb

related **0.11.0** doc:

user guide: https://analytics-zoo.readthedocs.io/en/v0.11.0/doc/Chronos/Overview/chronos.html

api doc: https://analytics-zoo.readthedocs.io/en/v0.11.0/doc/PythonAPI/Chronos/index.html

**Comment by MengleiZhang** [comment]
Method 1 works. Thank you.

**Comment by Shubhamshrimali** [comment]
![image](https://user-images.githubusercontent.com/20822359/135447143-e1d39d6e-fd13-43cc-984c-0f583d4ff114.png)

Hi I tried the above method but still not able install

**Comment by TheaperDeng** [comment]
> 
> 
> ![image](https://user-images.githubusercontent.com/20822359/135447143-e1d39d6e-fd13-43cc-984c-0f583d4ff114.png)
> 
> Hi I tried the above method but still not able install

![image](https://user-images.githubusercontent.com/35031544/135448401-006db21f-d203-4e7b-8a22-3c73913ba8e6.png)

The command seems to work just fine.

Some possible fixes are:
- check your network connection
- unset the 3rd party mirror source you may set for pip and use the default source
- download the whl directly and install by `pip install .[automl]` from [here](https://pypi.org/project/analytics-zoo/0.12.0b20210930/#files)

**Comment by Shubhamshrimali** [comment]
I am able to install other file like  tensorflow. so i can say there is no network issue 
![image](https://user-images.githubusercontent.com/20822359/135449253-cd61f298-853b-4a04-9305-0f64d5daaeb2.png)


unset the 3rd party mirror source you may set for pip and use the default source :-  i freshly build my system so i don't have any 3rd party mirror source 

download the whl directly and install by pip install .[automl] :-- 
![image](https://user-images.githubusercontent.com/20822359/135449766-22f3c468-b2a8-411f-9d8f-c478884221ce.png)

**Comment by TheaperDeng** [comment]
yeah I believe the issue is caused by platform conflict. analytics-zoo is tested throughly on mac and linux(or wsl on windows). Sadly we have not supported windows yet. The easiest way might be using WSL in windows.

For more details: 
https://analytics-zoo.readthedocs.io/en/latest/doc/UserGuide/python.html
https://analytics-zoo.readthedocs.io/en/latest/doc/Chronos/Overview/chronos.html
